cmd/bb_storage/main.go

```diff
@@ -85,10 +93,35 @@ func main() {
 	})
 
 	// RPC server.
-	s := grpc.NewServer(
-		grpc.StreamInterceptor(grpc_prometheus.StreamServerInterceptor),
-		grpc.UnaryInterceptor(grpc_prometheus.UnaryServerInterceptor),
-	)
+	var options []grpc.ServerOption
+	if len(*tlsCertFile) > 0 && len(*tlsKeyFile) > 0 {
```

Comment:
Maybe this is just personal preference, but I think it often makes more sense to use `*tlsCertFile != ""` for string comparison in Go. It's not only shorter, it also makes it more explicit this is about strings; not lists of objects.